value-resolver/src/test/kotlin/com/ritense/valueresolver/ValueResolverFactoryServiceTest.kt

```diff
@@ -79,7 +78,27 @@ internal class ValueResolverFactoryServiceTest {
     fun `Should throw exception on unknown prefix`() {
         val exception = assertThrows<RuntimeException> {
             resolverService.resolveValues(
-                processInstanceId = CamundaProcessInstanceId(UUID.randomUUID().toString()),
+                processInstanceId = UUID.randomUUID().toString(),
+                variableScope = DelegateTaskFake()
+                    .withVariable("firstName", "John")
+                    .withVariable("lastName", "Doe")
+                    .withVariable("active", true),
+                listOf(
+                    "xyz:firstName"
+                )
+            )
+        }
+
+        assertThat(exception.message).startsWith("No resolver factory found for value prefix xyz")
+    }
+
+    @Test
+    @Throws(RuntimeException::class)
+    fun `Should throw exception when no resolvers are configured`() {
```

Comment:
@marijnritense I added this to test the behaviour when no resolvers are found, as I don't think the plugin service is responsible for this.